sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -593,22 +596,37 @@ private void shutdownConsumerThreads() {
   }
 
   public interface FinalBuilder<T extends Configuration> {
+
+    /**
+     * Disables the health check for Kafka
+     *
+     * @return the same builder instance
+     */
+    FinalBuilder<T> withoutHealthCheck();
```

Comment:
Should we add some documentation in the module's README?

Reply:
Should we extend the Javadoc when this should be used and when not?

Reply:
Added documentation to both README and Javadoc.